### PR TITLE
Security/elk hardening

### PR DESCRIPTION
**Pull request by TimothyHou**

### CODE REVIEW

File: microservicecloud/bundle-ssl/elasticsearch/elasticsearch.crt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
+MIIDUjCCAjqgAwIBAgIUGr+NJMpgVN5QuI7iQRvi2t5XDX8wDQYJKoZIhvcNAQEL
```
Comment (akashdktyagi) [inline microservicecloud/bundle-ssl/elasticsearch/elasticsearch.crt:2]: @TimothyHou Tim, just wanted to check, These are public key right?
@GoF-NUS-ISS/gof-core

File: microservicecloud/docker-compose.backup [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: "3"
+ersion: "3"
```
Comment (akashdktyagi) [inline microservicecloud/docker-compose.backup:1]: Small mistake here., Version name. But its benign. Will fix it, myself.
@TimothyHou

File: microservicecloud/bundle-ssl/elasticsearch/elasticsearch.crt (hunk 1 above)
Comment (TimothyHou) [reply microservicecloud/bundle-ssl/elasticsearch/elasticsearch.crt:2]: yes all public key

File: microservicecloud/docker-compose.backup (hunk 2 above)
Comment (TimothyHou) [reply microservicecloud/docker-compose.backup]: thanks for pointing it out!

**Pull request merged**